src/System.Runtime/tests/System.Runtime.Tests.csproj

```diff
@@ -173,6 +173,9 @@
       <Link>Common\System\RandomDataGenerator.cs</Link>
     </Compile>
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'!='netfx'">
```

Comment:
What did  I do wrong here?

Reply:
There is no netfx configuration listed in src\System.Runtime\tests\Configurations.props. So when building the tree for netfx, the build will pick netstandard instead. TargetGroup will be netstandard.
I believe you need to add netfx to the list in that file and it will be fine. I don't see any tests for TargetGroup==netstandard that would be affected by doing that. 
We generally have followed a pattern of naming test files specially that are .NET Core specific, like HashCodeTests.netcoreapp.cs.

Reply:
Should I maybe move this down to netcoreapp tests then?

Reply:
It'd be nice not to restrict to netcoreapp since that loses uapaot (.NET Native) coverage

Reply:
Ok, so netcoreapp and uapaot are ok? The next item group restricts to exactly that: https://github.com/dotnet/corefx/pull/25013/files/9690fa0ff187835111fabe7e08d1a3490fd36d15#diff-6223985b207d5111dc7d0c2a37cff4e7R179

Reply:
That's ok with me